src/node/ext/server.cc

```diff
@@ -242,7 +277,9 @@ NAN_METHOD(Server::TryShutdown) {
     return Nan::ThrowTypeError("tryShutdown can only be called on a Server");
   }
   Server *server = ObjectWrap::Unwrap<Server>(info.This());
+  TryShutdownOp *op = new TryShutdownOp(server, info.This());
```

Comment:
The issue is that `TryShutdown` already uses the tag callback for the callback that the user passes to the function. That's why I needed the `OnComplete` op callback to do my own cleanup. But after you said this, I realized I could use the `OnComplete` op callback for `ForceShutdown` too.